src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx

```diff
@@ -80,6 +80,7 @@ export class CronJobTriggerDialog extends Component<CronJobTriggerDialogProps> {
       }, {
         spec: cronjobDefinition.spec.jobTemplate.spec,
         metadata: {
+          annotations: { "cronjob.kubernetes.io/instantiate": "manual" } as { [annotation: string]: string },
```

Comment:
This cast should not be necessary. Is it?

Reply:
Without the cast, `make build` throws an error (`make lint` and `make test` pass ok). 

![CleanShot 2022-03-07 at 13 08 05](https://user-images.githubusercontent.com/7034429/157072201-f36e0afb-7f87-4a3e-9683-56deb8cf9fcc.png)